sdk/src/main/java/com/relaypro/sdk/Relay.java

```diff
@@ -1415,6 +1412,13 @@ public static String getSourceUri(StartEvent startEvent) {
         if (object instanceof String) {
             sourceUri = (String) object;
         }
+        else if (object instanceof ArrayList){
```

Comment:
How about if you add a comment here for the Jira ticket number that this code is a workaround for? Then when iBot gets fixed, we can remove this code.